Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/TextToolbar.cs

```diff
@@ -36,9 +36,9 @@ public partial class TextToolbar : Control
         internal const string HeadersElement = "Headers";
 
         /// <summary>
-        /// Gets access to Generic Buttons that activate Formatter Methods
+        /// Gets access to Generic Buttons to instantiate for Creating Formatters (does not get the Current Instance from Toolbar)
         /// </summary>
-        internal CommonButtons CommonButtons { get; }
+        public CommonButtons CommonButtons { get; }
```

Comment:
Since this is just used internally and in the special formatter scenario (which contains the texttoolbar already), would it make sense for CommonButtons to just be a factory class anyway which takes in the model?